docs/css-prop.md

```diff
@@ -32,27 +78,37 @@ render(
 )
 ```
 
-You can also pass objects directly to the css prop, for more usage with objects, look at [the object styles page](/docs/object-styles.md).
+## Style Precedence
 
-```jsx
-// @live
+The `className` prop value overrides the styles provided to the `css` prop which is an important distinction when passing class names generated by emotion. Emotion removes generated class names from the `className` prop value, their underlying styles composed on top of the styles supplied to the `css` prop generating a new, single class name. The precedence order may seem counter-intuitive, but it allows developers predictably compose components and their styles.
```

Comment:
"Emotion removes generated class names from the `className` prop value"

should be

Emotion removes class names generated by itself from the `className` prop value

**I think**